adaptors/commcare.md

```diff
@@ -1,219 +1,204 @@
-best for mobile case management.
+[CommCare](https://www.dimagi.com/commcare/) is a mobile digital platform for
+data collection, case management, and service delivery, primarily used by
+organizations working in low-resource settings. CommCare provides a no-code
+interface that allows non-technical users to build customized mobile
+applications for various purposes, including surveys, client tracking, decision
+support for frontline workers, and service delivery monitoring.
 
-### Data Model
+CommCare boasts of some standout features that makes it a preferred choice for
+many organizations such as:
 
-CommCare data is primarily stored in **forms** and **cases**. Forms are the
-building blocks of applications and cases are used to track data on objects,
-usually people. Learn more about CommCare forms and cases at the links below.
+- Offline capability: Apps work offline, syncing data when connectivity is
+  available
+- Multi-platform: Runs on Android devices, feature phones, and web browsers
+- Multimedia support: Incorporates video, audio, images, and GPS data
+- Multilingual: Supports multiple languages within a single app
+- Security: Offers HIPAA, GDPR, and SOC2 compliant data storage and encryption
 
-- Case management:  
-  https://confluence.dimagi.com/display/commcarepublic/CommCare+Fundamentals+-+Case+Management
-- Form and case data in CommCare:
-  https://confluence.dimagi.com/display/commcarepublic/CommCare+Fundamentals+-+Data+in+CommCare
+#### Some common use cases where CommCare is adopted
 
-## Integration Use Cases
+- Community Health Worker: Collects data on health status, disease outbreaks,
+  and provides health education.
+- NGO: Monitors program outcomes, distributes surveys, and collects feedback
+  from beneficiaries.
+- Government: Collects data for public services, such as school enrollment,
+  health records, and census data.
 
-Example user stories:
+### How does OpenFn integrate with CommCare?
 
-- As a community health worker I want to store patient data that was collected
-  in rural areas with no internet access in Salesforce so I can better analyze
-  the data being collected and make more informed recommendations.
-- As a teacher I want to set up automatic message alerts to my students in order
-  to increase participation.
+OpenFn provides a CommCare adaptor that allows for seamless integration between
+CommCare and various external systems. This adaptor supports both data
+forwarding (push) and data fetching (pull) capabilities, enabling powerful data
+synchronization and integration scenarios.
 
-## APIs & Integration Options
+#### Some use cases include:
 
-CommCare offers a number of integration options for extracting and/or loading
-data to and from CommCare HQ.
+- Syncing data from CommCare to DHIS2: This integration allows for seamless
+  transfer of health data collected in CommCare to DHIS2 for comprehensive
+  analysis, visualization and reporting.
 
```

Comment:
Suggestions to add:
- Syncing data from CommCare to a Shared Health Repository: Health facilities that use CommCare as an EMR can integrate with national shared health repositories to send individual patient data.